spec/factories/case_log.rb

```diff
@@ -52,11 +52,11 @@
       reservist { 0 }
       illness { 1 }
       preg_occ { 2 }
-      tenancy_code { "BZ757" }
+      tenancy_code { Faker::Alphanumeric.alphanumeric(number: 10) }
```

Comment:
Tenancy code is not used in our application, it should be tenant code or we will need to update the form configuration.

Note: Having a random value will break all the export logic which expects pre-determined values. We will need to think of a better strategy to make this work.

Reply:
_I can find a log by entering a tenancy code_ - this is from requirement on the card. I made change to the test and it passes it.